### ISSUE TITLE
Caret ranges in npm specifiers do not work properly

### ISSUE THREAD

**Issue by masnagam** [hasundue/molt#112]
[caret ranges](https://github.com/npm/node-semver?tab=readme-ov-file#caret-ranges-123-025-004) seem not to work properly:

`molt` updates the version in the following import declaration:

```js
import ora from 'npm:ora@^7.0.1';
```

like this:

```
📦 ora ^7.0.1 => 8.0.1
...
```

i reproduced this issue with:

* deno 1.39.2
* molt 0b14e3e820ff669692041fb451041538199a263f
* on arch linux

**Comment by hasundue** [comment]
See also #111

**Comment by hasundue** [comment]
Fixed with #128.
Tilde or caret ranges are no longer targeted by molt.
